templates/taxi/car_detail.html

```diff
@@ -3,20 +3,38 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <p><span style="font-weight: bold">Manufacturer:</span> ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})
+  </p>
+  <form>
+    {% csrf_token %}
+
+    <h1>
+      Drivers
+      {% if user in car.drivers.all %}
+        <button type="submit" class="btn btn-danger link-to-page">
+          Delete me from this car
+        </button>
+      {% else %}
+        <button type="submit" class="btn btn-success link-to-page">
+          Assign me to this car
+        </button>
+      {% endif %}
+    </h1>
+  </form>
   <hr>
   <ul>
     {% for driver in car.drivers.all %}
```

Comment:
Similarly, `car.drivers.all` should be `car.drivers.all()` to correctly iterate over the queryset of drivers.